iopipe/contrib/logger/plugin.py

```diff
@@ -78,3 +86,9 @@ def post_report(self, report):
                         if 'uploads' not in plugin:
                             plugin['uploads'] = []
                         plugin['uploads'].append(signed_request['jwtAccess'])
+                if self.use_tmp is True:
+                    self.handler.stream.close()
```

Comment:
Does this `stream.close` delete the file? I see the docs say it deletes the file after each invocation

Reply:
Yes, `NamedTemporaryFile` is deleted when closed.